<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: 2015 Teo Mrnjavac &lt;teo@kde.org&gt;
SPDX-License-Identifier: GPL-3.0-or-later
 </author>
 <class>Calamares::DebugWindow</class>
 <widget class="QWidget" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="globalStorageTab">
      <attribute name="title">
       <string>GlobalStorage</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="globalStorageView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="jobQueueTab">
      <attribute name="title">
       <string>JobQueue</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="jobQueueText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modulesTab">
      <attribute name="title">
       <string>Modules</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="modulesListView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="modulesVerticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="typeLabel">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="moduleTypeLabel">
             <property name="text">
              <string>none</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="interfaceLabel">
             <property name="text">
              <string>Interface:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="moduleInterfaceLabel">
             <property name="text">
              <string>none</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="moduleConfigView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="crashButton">
       <property name="toolTip">
        <string>Crashes Calamares, so that Dr. Konqi can look at it.</string>
       </property>
       <property name="text">
        <string notr="true">Crash now</string>
       </property>
       <property name="icon">
        <iconset theme="data-error"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reloadStylesheetButton">
       <property name="toolTip">
        <string>Reloads the stylesheet from the branding directory.</string>
       </property>
       <property name="text">
        <string>Reload Stylesheet</string>
       </property>
       <property name="icon">
        <iconset theme="preferences-web-browser-stylesheets"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="widgetTreeButton">
       <property name="toolTip">
        <string>Displays the tree of widget names in the log (for stylesheet debugging).</string>
       </property>
       <property name="text">
        <string>Widget Tree</string>
       </property>
       <property name="icon">
        <iconset theme="view-list-tree"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendLogButton">
       <property name="toolTip">
        <string>Uploads the session log to the configured pastebin.</string>
       </property>
       <property name="text">
        <string>Send Session Log</string>
       </property>
       <property name="icon">
        <iconset theme="upload-media"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
